README.md

```diff
@@ -118,8 +118,33 @@ Both CI systems build all merges to master as well as pull requests.
 The repository contains a script [`setupLocalProxy.sh`](./scripts/setupLocalProxy.sh) which can be used to run the Proxy locally.
 A prerequisite for this is access to https://console.rh-idev.openshift.com/.
 To run the script you need to export your OpenShift access token for console.rh-idev.openshift.com as DSAAS_PREVIEW_TOKEN.
-Note, you need edit permissions on the dsaas-preview namespace in order to port forward.
-
+Note, you need edit permissions on the dsaas-preview namespace in order to port forward. 
+You will also need jq to run these commands. For fedora use `sudo dnf install jq`
+
+Usage: ./scripts/setupLocalProxy.sh [start|stop|env|unset]
+
+This script is used to run the Jenkins Proxy on localhost.
+As a prerequisite OPENSHIFT_API_TOKEN needs to be exported.
+In your shell (from the root of fabric8-jenkins-proxy):
+
+To start proxy and other required services
+```
+[user@localhost ~]$ export DSAAS_PREVIEW_TOKEN=<dsaas-preview-token> 
+[user@localhost ~]$ ./scripts/setupLocalProxy.sh start 
+[user@localhost ~]$ eval $(./scripts/setupLocalProxy.sh env) 
+[user@localhost ~]$ fabric8-jenkins-proxy 
+```
+
+To stop proxy and other services started for running it
```

Comment:
Can you please rephrase here?

Reply:
How about ` To stop running proxy, remove postgres container and stop port-forwarding to prod-preview's idler service and tenant service` ?

Reply:
perfect thanks just add the the in "To stop running *the* proxy, "